src/SingleCellAnnotator.imjoy.html

```diff
@@ -847,9 +838,40 @@
     },
     startRedrawAnnotation() {
       this.current_image_data.mask_adjusted = true;
-      this.vector_source.clear()
+      if (this.current_task_name !== 'segmentation'){
```

Comment:
Instead of checking the task name here, I think it's better to set a global state `single_cell_mode=true/false`, and use that to change the behaviour.